package.json

```diff
@@ -11,8 +11,8 @@
     "clean": "rm -rf ./packages/*/lib ./packages/*/*.tsbuildinfo",
     "build": "lerna run build",
     "build:docs": "lerna run build:refdocs && ./scripts/prepare-docs.sh",
-    "build:lib:watch": "lerna run build:lib:watch --parallel",
-    "build:types:watch": "lerna run build:types:watch --parallel",
```

Comment:
@nazarhussain what was the reason for removing `--parallel`? Looks like it is required to run multiple concurrent jobs
- see https://github.com/ChainSafe/lodestar/pull/5531